testApp/ios/DidomiExample.xcodeproj/project.pbxproj

```diff
@@ -966,7 +966,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
are these changes and the changes to the Podfile related to the change on the runner?

Reply:
It was part of the changes done while trying to fix `macos-latest` runner. I kept this one because it only affects the test app, and will be necessary at some point anyway.